pkg/dao/repository.go

```diff
@@ -1,22 +1,19 @@
 package dao
 
 import (
-	"fmt"
-
 	"github.com/content-services/content-sources-backend/pkg/api"
 	"github.com/content-services/content-sources-backend/pkg/db"
 	"github.com/content-services/content-sources-backend/pkg/models"
+	"github.com/jackc/pgconn"
 )
 
 type repositoryDaoImpl struct {
 }
 
-func GetRepositoryDao() repositoryDaoImpl {
+func GetRepositoryDao() RepositoryDao {
```

Comment:
I changed this return value to be the interface, that way other things could implement the interface and the handler code wouldn't care.